crates/sui-framework/packages/bridge/sources/bridge.move

```diff
@@ -537,6 +537,26 @@ module bridge::bridge {
         TRANSFER_STATUS_PENDING
     }
 
+    public fun get_token_transfer_action_signatures(
```

Comment:
is it ok to have this public function?
it seems so odd...

Reply:
changed this to private